src/test/java/com/wikia/webdriver/common/core/Helios.java

```diff
@@ -71,24 +71,19 @@ public static void deleteAllTokens(User user) {
     httpDelete.setHeader("THE-SCHWARTZ", Configuration.getCredentials().apiToken);
     httpDelete.setHeader("X-Wikia-Internal-Request", "0");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "X-Wikia-Internal-Request" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)